examples/sunpy_other_packages/reprojection_aia_euvi_mosaic.py

```diff
@@ -185,6 +185,10 @@
 outmap = sunpy.map.Map((array, header))
 outmap.plot_settings = maps[0].plot_settings
 
+outmap.nickname = 'AIA + EUVI/A + EUVI/B'
+outmap.meta.update({'wavelnth': maps[0].meta['wavelnth']})
```

Comment:
these should be moved into the header generation above.

Reply:
Done.